supervisor/container.go

```diff
@@ -29,21 +29,32 @@ type Container struct {
 	ownerPod *HyperPod
 }
 
-func (c *Container) run(p *Process) {
-	go func() {
-		err := c.create(p)
-		if err != nil {
-			c.ownerPod.sv.reap(c.Id, p.Id)
-			return
-		}
+func (c *Container) start(p *Process) error {
+	err := p.setupIO()
+	if err != nil {
+		return err
+	}
 
-		res := c.ownerPod.vm.WaitProcess(true, []string{c.Id}, -1)
-		if res == nil {
-			c.ownerPod.sv.reap(c.Id, p.Id)
-			return
-		}
+	res := c.ownerPod.vm.WaitProcess(true, []string{c.Id}, -1)
+	if res == nil {
+		c.ownerPod.sv.reap(c.Id, p.Id)
+		return fmt.Errorf("Failed to prepare waiting on the container")
+	}
 
-		err = c.start(p)
+	err = c.ownerPod.vm.Attach(p.stdio, c.Id)
+	if err != nil {
+		glog.V(1).Infof("Start Containter fail: fail to set up tty connection.\n")
```

Comment:
I think it's better to use  `glog.Errorf`, and reserve `glog.V(1)` as INFO, and `glog.V(3)` as DEBUG